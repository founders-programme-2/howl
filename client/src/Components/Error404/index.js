import React, { Fragment } from 'react';

const Error404 = () => {
  return (
    <Fragment>
      <h1>404!</h1>
      <h2>Sorry, the page you are looking for is unavailable.</h2>
    </Fragment>
  );
};

export default Error404;
