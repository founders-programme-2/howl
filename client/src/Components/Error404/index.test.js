import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Error404 from './index';

test('Natural', () => {
  const error404 = renderer.create(<BrowserRouter> <Error404 /> </BrowserRouter>).toJSON();
  expect(error404).toMatchSnapshot();
});
