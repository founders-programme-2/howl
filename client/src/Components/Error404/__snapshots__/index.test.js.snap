// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Natural 1`] = `
Array [
  " ",
  <h1>
    404!
  </h1>,
  <h2>
    Sorry, the page you are looking for is unavailable.
  </h2>,
  " ",
]
`;
